packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -497,33 +497,19 @@ export default class MeetingRequest extends StatelessWebexPlugin {
   }
 
   /**
-   * Make a network request to acknowledge a meeting
+   * Make a network request to record: start, stop, resume, or pause
    * @param {Object} options
-   * @param {String} options.locusUrl
-   * @param {String} options.deviceUrl
-   * @param {String} options.id
+   * @param {Object} options.body
+   * @param {String} options.uri
+   * @param {String} options.method
    * @returns {Promise}
    */
-  recordMeeting(options: {
-    locusUrl: string;
-    deviceUrl: string;
-    id: string;
-    recording: any;
-    paused: any;
-  }) {
-    const uri = `${options.locusUrl}/${CONTROLS}`;
-    const body = {
-      record: {
-        recording: options.recording,
-        paused: options.paused,
-      },
-    };
-
+  recordMeeting(options: {uri: string; body: object; method: string}): Promise<any> {
```

Comment:
This method is redundant now, it adds no value and has nothing specific about recording in the implementation. You could call it with a URL and body that would do something completely different like sending an SDP and it would also work, so I think we should just remove it.